index.js

```diff
@@ -17,9 +15,12 @@ const { roomInit, room } = require('./commands/room');
 const { contribute } = require('./commands/contribute');
 const { help } = require('./commands/help');
 
+const {aliasLightning} = require('./utils/aliasLightning');
```

Comment:
space after the brace - see the other imports.

also, this should still be in the `shared.js` file.

Reply:
did you mean move the aliaslightning.js to sharedjs?

Reply:
yes. there's no reason to have a utils folder with so few utils at this time.